freshclam/freshclam.c

```diff
@@ -191,6 +191,8 @@ static void help(void)
     printf("    --no-dns                             Force old non-DNS verification method\n");
     printf("    --checks=#n          -c #n           Number of checks per day, 1 <= n <= 50\n");
     printf("    --datadir=DIRECTORY                  Download new databases into DIRECTORY\n");
+    printf("                                         NOTE: DIRECTORY must already exist, be an absolute path, and");
+    printf("                                         be writeable by freshclam and readable by clamd/clamscan.");
```

Comment:
I'm good with this.  

Can you also add this to:
- the freshclam manpage:  https://github.com/Cisco-Talos/clamav/blob/main/docs/man/freshclam.1.in#L63-L64
- the freshclam.conf.sample:  https://github.com/Cisco-Talos/clamav/blob/main/etc/freshclam.conf.sample#L10-L13
- the freshclam.conf manpage:  https://github.com/Cisco-Talos/clamav/blob/main/docs/man/freshclam.conf.5.in#L64-L65

I also noticed that the comments in `freshclam.conf.sample` have more detail than in `freshclam.conf.5.in` manpage. If you can sync them (for this particular option), that would be nice.

Reply:
done

Reply:
done